src/scripts/modules/media/body/body.coffee

```diff
@@ -8,7 +8,7 @@ define (require) ->
   EditableView = require('cs!helpers/backbone/views/editable')
   ProcessingInstructionsModal = require('cs!./processing-instructions/modals/processing-instructions')
   SimModal = require('cs!./embeddables/modals/sims/sims')
-  Coach = require('cs!./embeddables/coach')
+  #Coach = require('cs!./embeddables/coach')
```

Comment:
Why was this commented? I _think_ this code is still actively being used. /cc @pandafulmanda